ios/gamma/split-view/RNSSplitViewHostComponentView.mm

```diff
@@ -183,6 +187,29 @@ - (void)updateProps:(const facebook::react::Props::Shared &)props
   [super updateProps:props oldProps:oldProps];
 }
 
+- (void)finalizeUpdates:(RNComponentViewUpdateMask)updateMask
+{
+  // Controller needs to know about the number of reactSubviews before its initialization to pass proper number of
+  // columns to the constructor. Therefore, we must delay it until the 1st transaction will be processed.
+  // At this point, children are already attached to the Host component, therefore, we may create SplitView controller
+  // right before 1st props update.
```

Comment:
```suggestion
  // right after 1st props update.
```

Reply:
We could move this to `didMoveToWindow`! Move whole controller initialization to separate method. Also I don't like allocating whole array of references just to count number of columns. Let's just do a simple `for` with counter (could also be in separate helper method).

Reply:
https://github.com/software-mansion/react-native-screens/pull/3003/commits/dcb532483489d2189ea06dc304928fe49eafbbd2